test/http_binding_1.ts

```diff
@@ -35,8 +35,7 @@ const cloudevent = new CloudEvent({
   dataschema,
   data,
 });
-cloudevent[ext1Name] = ext1Value;
-cloudevent[ext2Name] = ext2Value;
+cloudevent = cloudevent.cloneWith({ [ext1Name]: ext1Value, [ext2Name]: ext2Value });
```

Comment:
Why can't we just add this to the original CloudEvent?

Reply:
probably could.  Looks like a leftover of when you couldn't add extensions during construction